config.go

```
@@ -1022,6 +1024,46 @@ func ValidateConfig(cfg Config, usageMessage string,
 			numNets++
 			cfg.ActiveNetParams = chainreg.BitcoinSimNetParams
 		}
+		if cfg.Bitcoin.SigNet {
```

Let's return an error if signet is configured for litecoin?

---

Good idea, done.